.golangci-kal.yml

```diff
@@ -16,6 +16,7 @@ linters:
             enable:
               - "commentstart" # Ensure comments start with the serialized version of the field name.
               - "conditions" # Ensure conditions have the correct json tags and markers.
+              - "duplicatemarkers" # Ensure exact duplicates of markers for types and fields.
```

Comment:
In which sense "exact"? 

e.g. what if we have two enum markers and they have different enum values? (I assume the linter would still pick it up, so maybe we should clarify waht exact means)

Reply:
The more I think about it the more I'm confused :)

Does it ensure there are no duplicate markers at all, or does it ensure that if there are duplicate markers they are exactly identical?

Reply:
The purpose of this one is to catch if you have, for example two `==` identical strings, eg if you had `+optional` twice.

We are going to build another rule that looks for markers that must be specified only once, which would pick up two copies of enum but with different values

Reply:
The meaning `exact` is that duplicatemarkers only diagnose complete match. If there are two enum markers on the type and field, then they have different values, it doesn't pick up since the Enum value is different.
The case you say will be picked up by the uniquemarkers we're currently implementing.

@JoelSpeed 
If I am misleading or missing something, I welcome your comments.

Reply:
It also picks up from field and corresponding type. So if you had `MaxLength` identically at the field and at the type definition, it would suggest to remove the one at the field to avoid duplication.

In theory this is more important when the values differ, so that's where a second rule/update to this rule will come in to make sure you'd only have one value for `MaxLength` across field and type to prevent the schema being generated with a conflicting choice

Reply:
Okay, got it. Can we update the godoc comment to reflect that it ensure that we *don't* have exact duplicate markers? (if think the current comment is inverted)